monocle/github/pullrequest.py

```diff
@@ -296,6 +297,7 @@ def extract_pr_objects(pr):
             change = {}
             change['type'] = 'Change'
             change['id'] = pr['id']
+            change['draft'] = pr['isDraft']
```

Comment:
You need to add that new field in the schema.

Reply:
Could you add an issue to add the same support in Gerrit ?

Reply:
is the same feature available in gerrit?

Reply:
Seems there was a draft support that has been removed. There is a "work in progress"  https://www.gerritcodereview.com/2.15.html#support-for-draft-changes-removed
Well let's manage that if when someone will require it.

Reply:
In https://github.com/morucci/monocle/blob/master/monocle/db/db.py#L101 add "draft": {"type": "boolean"},